packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -1065,11 +1081,20 @@ contract Deploy is Deployer {
                     _blockOracle: BlockOracle(mustGetAddress("BlockOracle"))
                 })
             );
-
```

Comment:
^ Above, we need to switch between the `FaultDisputeGame` and the `OutputBisectionGame`